### PR TITLE
OCD-4379 - Add ROLE_ONC, ROLE_DEVELOPER, ROLE_ACB

### PR DESCRIPTION
**Pull request by tmy1313**

### CODE REVIEW

File: src/app/components/change-request/change-request-history.jsx [hunk 1]
```diff
@@ -42,13 +42,13 @@ function ChplChangeRequestHistory(props) {
       };
       switch (item.userPermission.authority) {
         case 'ROLE_ADMIN':
-        case 'ROLE_ONC':
+        case 'chpl-onc':
```
Comment (andlar) [inline src/app/components/change-request/change-request-history.jsx]: I don't think this should be updated. Until the change request stuff is changed the data that's stored in the CR data will still have the old style `ROLE_` stuff. I don't know what we're going to need to change this to later, but it should be in a different path

File: src/app/components/change-request/change-request-history.test.jsx [hunk 2]
```diff
@@ -10,7 +10,7 @@ const mock = {
   changeRequest: {
     statuses: [{
       id: 1,
-      userPermission: { authority: 'ROLE_ONC' },
+      userPermission: { authority: 'chpl-onc' },
```
Comment (andlar) [inline src/app/components/change-request/change-request-history.test.jsx]: Same comment wrt CR data

File: src/app/components/change-request/change-request.test.jsx [hunk 3]
```diff
@@ -19,7 +19,7 @@ const mock = {
   changeRequest: {
     statuses: [{
       id: 1,
-      userPermission: { authority: 'ROLE_ONC' },
+      userPermission: { authority: 'chpl-onc' },
```
Comment (andlar) [inline src/app/components/change-request/change-request.test.jsx]: Similar note

File: src/app/components/user/user-invite.test.jsx [hunk 4]
```diff
@@ -11,7 +11,7 @@ const hocMock = {
   dispatch: jest.fn(),
 };
 
-const rolesMock = ['ROLE_ADMIN', 'ROLE_ONC'];
+const rolesMock = ['ROLE_ADMIN', 'chpl-onc'];
```
Comment (andlar) [inline src/app/components/user/user-invite.test.jsx]: If the `ROLE_ADMIN` one is still here, should we be converting the `ROLE_ONC` one?

**Review by andlar — CHANGES_REQUESTED** [review]

**Review by kekey1 — DISMISSED** [review]

**Review by kekey1 — DISMISSED** [review]

**Review by andlar — DISMISSED** [review]

**Review by andlar — DISMISSED** [review]

**Review by kekey1 — DISMISSED** [review]

**Review by kekey1 — APPROVED** [review]

**Review by andlar — APPROVED** [review]

**Pull request merged**